build.gradle.kts

```diff
@@ -13,6 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+//import de.fayard.OrderBy
```

Comment:
Planning to delete here and below?

Reply:
Oh, i still want to try to get this working so we don't have to keep manually sorting that file.

Reply:
Deleted instead, don't want to fight with gradle more than absolutely necessary today.